modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseService.java

```diff
@@ -21,38 +21,67 @@
  */
 public enum CouchbaseService {
 
-  /**
-   * Key-Value service.
-   */
-  KV("kv"),
-
-  /**
-   * Query (N1QL) service.
-   */
-  QUERY("n1ql"),
-
-  /**
-   * Search (FTS) service.
-   */
-  SEARCH("fts"),
-
-  /**
-   * Indexing service (needed if QUERY is also used!).
-   */
-  INDEX("index"),
-
-  /**
-   * Analytics service.
-   */
-  ANALYTICS("cbas");
-
-  private final String identifier;
-
-  CouchbaseService(String identifier) {
-    this.identifier = identifier;
-  }
-
-  String getIdentifier() {
-    return identifier;
-  }
+    /**
+     * Key-Value service.
+     */
+    KV("kv", 256),
```

Comment:
I was wondering if it is really a good thing to hardcode the minimum quotas. Couldn't these be even image/version dependent?

Reply:
@kiview while in theory the minimum quotas can change, they have been pretty steady and are enforced in the UI and on the server .. 

I think we could remove the runtime check for smaller, but then it will fail with a server error instead (and might be more confusing?) (https://docs.couchbase.com/server/current/learn/buckets-memory-and-storage/memory.html#service-memory-quotas)

Reply:
I would trust your decision as the technology expert in this case. The only thing we should avoid are situations, where users end up in an unresolvable situation on their side (having to absolutely set a value, that we don't allow in the code).

Reply:
Yeah, I guess it makes sense. I'll follow up with another PR on this once we've got the current 3 PRs in so that it doesn't get too confusing.